sensor/kubernetes/listener/resources/route_test.go

```diff
@@ -160,6 +167,9 @@ func (suite *RouteAndServiceDispatcherTestSuite) TestServiceCreateWithPreexistin
 }
 
 func (suite *RouteAndServiceDispatcherTestSuite) TestManyRoutesMatchingAndDeletions() {
+	if features.ResyncDisabled.Enabled() {
```

Comment:
Add todo linking to task that removes old code path.

Reply:
Done